scripts/upgrade

```diff
@@ -72,82 +66,90 @@ ynh_clean_setup () {
 # Exit if an error occurs during the execution of the script
 ynh_abort_if_errors
 
+#=================================================
+# CHECK THE PATH
+#=================================================
+
+# Normalize the URL path syntax
+path_url=$(ynh_normalize_url_path $path_url)
+
 #=================================================
 # STANDARD UPGRADE STEPS
 #=================================================
 # DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
+ynh_print_info "Upgrading source files..."
 
-# Move old app dir
-mv "$final_path" "$final_path.old"
-
+# Download, check integrity, uncompress and patch the source from app.src
 ynh_setup_source "$final_path"
-mkdir -p "$final_path"/sessions
 
-# restore data
-cp -a "$final_path.old/data" "$final_path"
```

Comment:
Why did you delete that part?

Reply:
Duplicating the directory isn't needed. We can overwrite the whole directory directly.
`data` and `plugins` does not contains files in the archive that would delete anything in an instance already used.

Reply:
Ok!